pbj-core/pbj-runtime/src/main/java/com/hedera/pbj/runtime/io/buffer/BufferedData.java

```diff
@@ -55,7 +81,7 @@ private BufferedData(@NonNull final ByteBuffer buffer) {
      */
     @NonNull
     public static BufferedData wrap(@NonNull final ByteBuffer buffer) {
-        return new BufferedData(buffer);
+        return wrap(buffer, false);
```

Comment:
This can be direct or not direct depends on the ByteBuffer passed in. Can just ask the byte buffer.